games-engines/qtads/qtads-3.0.0.ebuild

```diff
@@ -0,0 +1,37 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI=7
+inherit qmake-utils xdg
+
+DESCRIPTION="Multimedia interpreter for TADS text adventures"
+HOMEPAGE="https://realnc.github.io/qtads"
+SRC_URI="https://github.com/realnc/qtads/releases/download/v${PV}/${P}-source.tar.xz"
+
+LICENSE="GPL-3+"
+SLOT="0"
+KEYWORDS="~amd64 ~x86"
+IUSE="+sound"
+
+DEPEND="dev-qt/qtnetwork:5[ssl]
+	dev-qt/qtwidgets:5
```

Comment:
I know it's somewhat implied but I think you should also have `dev-qt/qtcore` and `dev-qt/qtgui` here as they explicitly appear in the final link and `-Wl,--as-needed` doesn't prune them from the `NEEDED` entries.